var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -311,6 +311,15 @@ def flag_handler(self, name, flags):
         if (name == "cflags" or name == "cxxflags") and self.spec.satisfies("%intel"):
             flags.append("-no-ipo")
             return (None, None, flags)
+
+        if name in ("cflags", "cxxflags"):
+            if "+hdf5" in self.spec:
+                if "@:5.9" in self.spec:
```

Comment:
```suggestion
                if self.spec.satisfies("@:5.9"):
```

Reply:
I generally avoid using `satisfies` in `package.py`s, the `<range/variant> in <spec>` format feels more "python"-y and is the prevalent method in the packages I work on. Although, if there is a stronger reason to use satisfies, I am happy to change my thinking.

Reply:
just noticed there was a link to docs 🤦🏽